lnrpc/walletrpc/walletkit.proto

```diff
@@ -768,6 +768,10 @@ message FundPsbtRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 7;
+
+    // The time in seconds before the wallet's lock on the selected UTXOs expires.
+    // If set to zero, the default lock duration is used.
+    int32 utxo_lease_expiration_seconds = 8;
```

Comment:
Out of curiosity, what is the use case for this exactly?

What I run into myself, and I think this is related, is that I can't use FundPsbt without a double-spend risk. See also #5425. The problem is the automatic expiry of the internal leases. Changing the expiry time doesn't fix that. In my case, I'd be better off with the ability for `FundPsbt` to use externally locked utxos. That way, as the caller, I am in complete control over the leases.